src/events/request/RequestReactionRemovalEventHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { MessageReaction, User } from 'discord.js';
+import * as log4js from 'log4js';
+import EventHandler from '../EventHandler';
+
+export default class RequestReactionRemovalEventHandler implements EventHandler<'messageReactionAdd'> {
+	public readonly eventName = 'messageReactionAdd';
+
+	private logger = log4js.getLogger( 'RequestReactionRemovalEventHandler' );
+
+	// This syntax is used to ensure that `this` refers to the `RequestResolveEventHandler` object
+	public onEvent = async ( reaction: MessageReaction, user: User ): Promise<void> => {
+		this.logger.info( `User ${ user.tag } added '${ reaction.emoji.name }' reaction to request message '${ reaction.message.id }'` );
+                const guildMember = reaction.members.fetch(user);
+
+                if ( !guildMember.roles.cache.has( #mod role id ) && !guildMember.roles.cache.has( #helper role id ) ) {
```

Comment:
It would be great to move this part to config in form of a list of strings (probably called `privilegedUsers`?). You can get the role ID with Discord developer mode enabled, but anyways, `647812320629882910` is moderator, `647812604949037056` is helper, `647812889133973514` is mojang, and `647814138818723850` is bot.

Reply:
Another way would be to check if the user has the reaction permission, which is imo easier and better to implement.